packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -305,13 +306,19 @@ export class ElectronMainApplication {
 
     async openDefaultWindow(): Promise<BrowserWindow> {
         const [uri, electronWindow] = await Promise.all([this.createWindowUri(), this.createWindow()]);
+        if (!this.useNativeWindowFrame) {
```

Comment:
On Linux, I ended up with two menubars, so I got rid of the native one using this workaround. Maybe I should try and figure out why that happens instead of fixing the symptom :wink: